examples/openapi-tools/quickstart-se/se-server/src/main/java/org/openapitools/server/api/MessageServiceImpl.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2022 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.openapitools.server.api;
+
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.logging.Logger;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.helidon.common.http.Http;
+import io.helidon.webserver.ServerRequest;
+import io.helidon.webserver.ServerResponse;
+
+import org.openapitools.server.model.Message;
+
+public class MessageServiceImpl implements MessageService {
+
+    private final AtomicReference<Message> defaultMessage = new AtomicReference<>();
+
+    private static final int HTTP_CODE_NOT_IMPLEMENTED = 501;
+    private static final Logger LOGGER = Logger.getLogger(MessageService.class.getName());
+    private static final ObjectMapper MAPPER = JsonProvider.objectMapper();
+
+    public MessageServiceImpl() {
+        Message message = new Message();
+        message.setMessage("World");
+        message.setGreeting("Hello");
+        defaultMessage.set(message);
+    }
+
+    public void getDefaultMessage(ServerRequest request, ServerResponse response) {
+        response.send(defaultMessage.get());
+    }
+
+    public void getMessage(ServerRequest request, ServerResponse response) {
+        String name = request.path().param("name");
+        Message result = new Message();
```

Comment:
Here you used the suggested fix I mentioned in my previous review, but I like the fix you did in the MP example better:

```
        Message result = new Message();
        return result.message(defaultMessage.get().getMessage()).greeting(name);
```
Just make sure that the instructions in the `README.md` and the code in this example file are the same.

Reply:
Unfortunately, it is not possible because the class `Message` contains only standard getters and setters.

Reply:
Good point! (Our generators should probably all share the same model template but are clearly not.)